### PR TITLE
Fix account undefined not found

### LINKED ISSUE #911: Fix: Account undefined is not found…
It looks like there's a small issue as shown in this screenshot:

Screen Shot 2022-01-04 at 3 28 29 PM

You can reproduce by running:

    near view pretend.mike.testnet fake_method ''

It should not say "undefined" and instead say the name of the NEAR account.

### PR DESCRIPTION
**Pull request by narwalsandeep**
Fixes #911

### CODE REVIEW

**Comment by narwalsandeep** [comment]
@mikedotexe PR

File: utils/exit-on-error.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ process.on('exit', () => {
 });
 
 module.exports = (promiseFn) => async (...args) => {
+
```
Comment (willemneal) [inline utils/exit-on-error.js:35]:
```suggestion
```

Comment (willemneal) [reply utils/exit-on-error.js]: Thanks! I have no power here and @volovyk-s is the better bet when tagging for a review.

**Comment by narwalsandeep** [comment]
@volovyk-s plz review

**Pull request merged**